### PR TITLE
Issue #1567 Make CONTRIBUTING.adoc more discoverable

### PR DESCRIPTION
**Pull request by robin-owen**
Let me know if this isn't what was desired or if any changes need to take place.

Notes:

Linked as "Contribution guidelines" in order to be consistent with the header casing in **_CONTRIBUTING.adoc_**. Linked directly to the `#contributing-guidelines` anchor in order to make the guidelines immediately visible and clear when the link is visited.

### CODE REVIEW

**Comment by LalatenduMohanty** [comment]
@robin-owen Thanks for the PR. I think it is ok to have two references to the contributing doc. Will wait little bit see if others have different opinion. @minishift/minishift-dev

File: docs/source/contributing/index.adoc [hunk 1]
```diff
@@ -5,6 +5,7 @@ include::variables.adoc[]
 
 Join the {project} community! Learn about the link:https://github.com/minishift/minishift/blob/master/README.adoc[project] and how to link:https://github.com/minishift/minishift/blob/master/CONTRIBUTING.adoc[contribute].
 
+- link:https://github.com/minishift/minishift/blob/master/CONTRIBUTING.adoc#contribution-guidelines[Contribution guidelines]
```
Comment (budhrg) [inline docs/source/contributing/index.adoc]: only `Guidelines` ? as the page title is `Contributing to Minishift`.

Comment (LalatenduMohanty) [reply docs/source/contributing/index.adoc]: @budhrg Not sure what you mean. The PR is for making the contribution guideline visible to users.

Comment (budhrg) [reply docs/source/contributing/index.adoc]: Works for me as well.

**Pull request merged**